test/chat_api_web/controllers/twilio_controller_test.exs

```diff
@@ -0,0 +1,97 @@
+defmodule ChatApiWeb.TwilioControllerTest do
+  use ChatApiWeb.ConnCase
+
+  import Mock
+  import ChatApi.Factory
+
+  alias ChatApi.Twilio
+  alias ChatApi.Customers.Customer
+  alias ChatApi.Messages
+  alias ChatApi.Messages.Message
+  alias ChatApi.Conversations.Conversation
+  alias ChatApi.Twilio.TwilioAuthorization
+
+  setup %{conn: conn} do
+    account = insert(:account)
+    user = insert(:user, account: account)
+    conn = put_req_header(conn, "accept", "application/json")
+    authed_conn = Pow.Plug.assign_current_user(conn, user, [])
+
+    {:ok, conn: conn, authed_conn: authed_conn, account: account}
+  end
+
+  describe "webhook" do
+    @request_body %{
+      "AccountSid" => "1234",
+      "To" => "1234",
+      "From" => "1234",
+      "Body" => "body"
+    }
+
+    test "returns 200 when message is successfuly created",
+         %{authed_conn: authed_conn} do
+      with_mocks([
```

Comment:
I've decided to use mocks when testing in controller so that I don't have to insert actual data to the db. It seems like this makes sense because it separates the concerns between the controller vs context. The context should do all the business logic and the controller is responsible the web layer.